host-interaction/os/users/get-logon-sessions.yml

```diff
@@ -0,0 +1,16 @@
+rule:
+  meta:
+    name: get logon sessions
+    namespace: host-interaction/os/users
```

Comment:
we have `host-interaction/session` already. perhaps that's a better existing namespace for this rule?

Reply:
"when to use 'session' and when to use 'users'?" i don't know, but lets go for consistency right now :-)